hackathon/views.py

```diff
@@ -226,6 +328,9 @@ def update_hackathon(request, hackathon_id):
         if form.is_valid():
             form.instance.updated = now
             form.save()
+            # saved_form = form.save(commit=False)
```

Comment:
Do we need these still if commented-out?

Reply:
Removed commented out code.